python/dist2wheel.py

```diff
@@ -66,7 +66,7 @@ def create_all(self):
 
         # Burn a pass through the list to steal some useful bits from the Brew config
         for artifact in self.artifacts:
-            if "BrewConfig" in artifact["extra"]:
+            if "BrewConfig" in artifact.get("extra", {}):
```

Comment:
/cc @0xDEC0DE

Reply:
This should probably procure these values from elsewhere if it can't find them here.  Especially declaring the license being used.

Reply:
Yeah, I had considered that.  I'll likely come back and revisit that.  My latest push makes it a little less brittle.